lib/hexpm/web/templates/package/show.html.eex

```diff
@@ -73,6 +73,11 @@ tools = Keyword.take(tools, compatible_tools)
       <span class="license"><%= Enum.join(licenses, ", ") %></span>
     <% end %>
 
+    <%= if @current_release.checksum do %>
```

Comment:
we don't actually need an `if` because all releases have checksums:

```elixir
# prod
iex> from(r in Release, select: count(r.id), where: is_nil(r.checksum)) |> Repo.one
0
```

I'm gonna merge as is, and remove the `if` on master. I'm gonna also add a `NOT NULL` constraint for this field. Thank you!